File: MoonlightGarden/Framework/Utils/ApiResponse.cs
namespace MoonlightGarden.Framework.Utils
{
    public class ApiResponse
    {
        public bool success { get; set; }
        public object data { get; set; }
        public ApiResponse(object data)
        {
            this.success = data != null;
            this.data = data;
        }
    }
}

File: src/main/java/com/moonlightgarden/util/ApiResponse.java
package com.moonlightgarden.util;

public class ApiResponse {
  private boolean success;
  private Object data;
  
  public ApiResponse(Object data) {
    this.success = data != null;
    this.data = data;
  }

  public boolean isSuccess() {
    return success;
  }
  public void setSuccess(boolean success) {
    this.success = success;
  }
  public Object getData() {
    return data;
  }
  public void setData(Object data) {
    this.data = data;
  }
}
